.github/CONTRIBUTING.md

```diff
@@ -70,24 +70,57 @@ have a chance of keeping on top of things:
 
 ## Unit-test your changes
 
-Please cover all changes with unit tests and make sure that your code does not
-break any existing tests. We will only merge pull request that include full
+Please cover all changes with automatic tests and make sure that your code does
+not break any existing tests. We will only merge pull request that include full
 code coverage of the fixed bugs and the new features.
 
-To run the existing PHPUnit tests, run this command:
+### Running the unit tests
 
-    vendor/bin/phpunit Tests/
+To run the existing unit tests, run this command:
+
+    vendor/bin/phpunit -c Configuration/PHPUnit/phpunit.xml Tests/Unit/
+
+### Running the integration tests
+
+For being able to run the integration tests, you will need a local MySQL
+database and a user with access permissions to that database.
+
+After you have created the database and the user, please import the database
+schema once. Assuming that your database is named `phplist_test`, the user is
+named `phplist`, and the password is `batterystaple`, the command looks like
+this:
+
+    mysql -u phplist_test --password=batterystaple phplist_test < Database/Schema.sql
+
+When running the integration tests, you will need to specify the database name
+and access credentials on the command line (in the same line):
+
+    PHPLIST_DATABASE_NAME=phplist_test PHPLIST_DATABASE_USER=phplist PHPLIST_DATABASE_PASSWORD=batterystaple vendor/bin/phpunit -c Configuration/PHPUnit/phpunit.xml Tests/Integration/
 
 
 ## Coding Style
 
+Please make your code clean, well-readable and easy to understand.
+
 Please use the same coding style ([PSR-2](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md))
 as the rest of the code. Indentation for all files is four spaces.
 
 We will only merge pull requests that follow the project's coding style.
 
 Please check your code with the provided PHP_CodeSniffer standard:
 
-    vendor/bin/phpcs --standard=PSR2 Classes/ Tests/
+    vendor/bin/phpcs --standard=vendor/phplist/phplist4-core/Configuration/PhpCodeSniffer/ Classes/ Tests/
 
-Please make your code clean, well-readable and easy to understand.
+Please also check the code structure using PHPMD:
+
+    vendor/bin/phpmd Classes/ text vendor/phplist/phplist4-core/Configuration/PHPMD/rules.xml
+
+And also please run the static code analysis:
+
+    vendor/bin/phpstan analyse -l 5 Classes/ Tests/
+
+You can also run all code style checks using one long line:
```

Comment:
I propose specifying that this is bash shell specific; e.g. "You can also run all code style checks using one long line from a bash shell:"